Toivoa_Backend/Router/userRouter.js

```diff
@@ -1,30 +1,12 @@
-const express = require('express');
```

Comment:
Keep the routes intact.
https://github.com/Minimal-Alexi/ProjectWB/blob/sprint-3/Toivoa_Backend/Router/userRouter.js